RELEASE/scripts/autoscend/quests/level_09.ash

```diff
@@ -783,35 +784,23 @@ boolean L9_oilPeak()
 		return false;
 	}
 
-	if (contains_text(visit_url("place.php?whichplace=highlands"), "fire3.gif"))
-	{
-		if (!auto_is_valid($item[Bubblin\' Crude]))
-		{
-			return false;
-		}
+	if (contains_text(visit_url("place.php?whichplace=highlands"), "fire3.gif")) {
 		int oilProgress = get_property("twinPeakProgress").to_int();
-		boolean needJar = ((oilProgress & 4) == 0);
-
-		if((item_amount($item[Bubblin\' Crude]) >= 12) && needJar)
-		{
-			if(auto_my_path() == "G-Lover")
-			{
-				if(item_amount($item[Crude Oil Congealer]) == 0)
-				{
-					cli_execute("make " + $item[Crude Oil Congealer]);
+		boolean needJar = ((oilProgress & 4) == 0) && item_amount($item[Jar Of Oil]) == 0;
+		if (!needJar) {
+			return false;
+		} else if (item_amount($item[Bubblin' Crude]) >= 12) {
+			if (auto_my_path() == "G-Lover") {
+				if (item_amount($item[Crude Oil Congealer]) < 1) {
```

Comment:
also check that we have enough G in inventory to buy it. it costs 3 G to get oil congealer and on your first Glover run you will not have it. which i think will cause errors. 
actually... it should probably be checked before we collect the bubblin' crude. so in needJar?

maybe put something like
```
if(auto_my_path() == "G-Lover" && item_amount($item[G]) < 3)
{
needJar = false;
}
```